scss/tokens/_raw.scss

```diff
@@ -16,4 +16,36 @@ $ouds-opacity-500: .32 !default;
 $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .88 !default;
 $ouds-opacity-900: 1 !default;
-// scss-docs-end ouds-opacities-variables
+// scss-docs-end ouds-raw-opacity
+
+// Border
+// scss-docs-start ouds-raw-border
+$ouds-border-base:      4px !default;
+
+$ouds-border-width-0:   0 !default;
+$ouds-border-width-25:  $ouds-border-base * .25 !default;
+$ouds-border-width-50:  $ouds-border-base * .5 !default;
+$ouds-border-width-75:  $ouds-border-base * .75 !default;
+$ouds-border-width-100: $ouds-border-base !default;
+// $ouds-border-width-150: $ouds-border-base * 1.5 !default;
+// $ouds-border-width-200: $ouds-border-base * 2 !default;
+
+$ouds-border-radius-0:    0 !default;
+// $ouds-border-radius-25:   $ouds-border-base * .25 !default;
+// $ouds-border-radius-50:   $ouds-border-base * .5 !default;
+$ouds-border-radius-75:   $ouds-border-base * .75 !default;
+// $ouds-border-radius-100:  $ouds-border-base !default;
+$ouds-border-radius-150:  $ouds-border-base * 1.5 !default;
+// $ouds-border-radius-200:  $ouds-border-base * 2 !default;
+$ouds-border-radius-300:  $ouds-border-base * 3 !default;
+// $ouds-border-radius-400:  $ouds-border-base * 4 !default;
+// $ouds-border-radius-500:  $ouds-border-base * 5 !default;
+// $ouds-border-radius-600:  $ouds-border-base * 6 !default;
+// $ouds-border-radius-800:  $ouds-border-base * 8 !default;
+$ouds-border-radius-9999: 2000px !default;
```

Comment:
This one (pill) will be dropped from the tokens as said today during the workshop.
It can be handled like `"circle": 50%` in `$ouds-border-radiuses`.